shared/core-utils/ffprobe.ts

```diff
@@ -0,0 +1,180 @@
+import { ffprobe, FfprobeData } from 'fluent-ffmpeg'
```

Comment:
This file should not be in `core-utils` since it imports a non nodejs core module